Feature/value and reference type

BI003 opened the pull request.

BI003 committed:
Создана папка “Domain”, в неё добавлена “Библиотека
классов” - “BankSystem.Domain”, в неё добавлена папка “Models”.
Добавлены классы:
“Person” (человек),  “Employee” (сотрудник), “Client”
(клиент), и структура “Currency” (валюта).
В папку “Tools” добавлен проект (Консольное приложение) - “Practice”.

BI003 committed:
Добавлены конструкторы в класс "Person" и его дочерние классы.
В класс "Employee" добавлено поле "Contract".

BI003 committed:
Были созданы свойства классов и структуры, а так же методы:
"CreateContract" - для обновления контракта сотрудника (свойство "Contract", класса "Employee").
"Update" - для обновления значения свойства "Price", структуры "Cyrrency".

BI003 committed:
Были созданы:
Папка "Servies"
Проект "BankSystem.App"
Класс “BankService”.
В классе "BankService" были созданы поля и 2 метода:
"OwnerSalaryCalculation" - для вычисления зарплаты владельцев банка.
"ConvertClientInEmployee" - для преобразования клиента банка в сотрудника.

BI003 committed:
Метод "OwnerSalaryCalculation" был полность переработан.
Классы "BankService" и "Employee" были обновлены.

Dem-Ivan commented on BankSystem.App/Class1.cs:
```diff
@@ -0,0 +1,7 @@
+    public class Class1
+    {
+
+    }
```
Мусор

Dem-Ivan commented on BankSystem.Domain/Models/Client.cs:
```diff
@@ -0,0 +1,9 @@
+{
+    public class Client : Person
+    {
+        public Client(string name, string surname, int passport, int number) : base(name, surname, passport, number)
```
Убрать конструкторы из моделей.

BI003 committed:
Была выполнена работа над замечаниями, а конкретно переработан метод "OwnerSalaryCalculation" - для расчёта зарплаты владельцев.
В связи с этим были минимально скорректированны классы: "BankService" и "Employee".

BI003 commented:
Сливаю ветви после исправления замечаний.
